services/test-validators.js

```diff
@@ -91,6 +91,39 @@ const isDependencyState = withRegex(
   /^(\d+ out of date|\d+ deprecated|up to date)$/
 )
 
+const getTestTotalsValidator = ({ passed, failed, skipped }) =>
```

Comment:
Same comment as above, as both Appveyor and Azure DevOps Tests were replicating this same functional code in multiple places.

Instead of replicating it again for JenkinsTests, I figured it would be better to put it here in one place and consume it from the other service testers (if there's no opposition to this, I will open a future PR to update Appveyor and Azure DevOps test services to leverage this). We've also got some additional test badges that have been requested (for example CircleCI and Sonar) that can re-use this as well.

Reply:
Looks nice! Appreciate the cleanup, definitely makes this more enjoyable to read.

I've a slight preference to call these `makeTestTotalsValidator`.

Reply:
Renamed accordingly